CHANGELOG.md

```diff
@@ -111,6 +111,8 @@ file an issue, so that we can see how to handle this.
   ```
   There is also a convenience header `<FairBoostSerializationSupport.h>` which includes all
   class-based support headers currently offered.
+* Check the return value of `source->InitUnpackers()`/`source->ReinitUnpackers()`
+  in `FairRunOnline`. Stop run if `false` returned.
```

Comment:
As 18.8 was released already, either
* move this into the 19.0 section (if it only targets that release),
* or create a new 18.8.1 section and add it there (if you target 18.8.x)